### PR TITLE
Responder support for Get numeric effecter command

### PR DESCRIPTION
**Pull request by ArchanaKakani**
Added responder support for GetNumericEffecter Command

Test:
Using pldmtool raw command
    pldmtool raw -d 0x80 0x02 0x32 0x23 00
    pldmtool: Tx: 08 01 80 02 32 23 00
    pldmtool: Rx: 08 01 00 02 32 00 00 01 03 03

### CODE REVIEW

File: libpldmresponder/platform_numeric_effecter.hpp [hunk 1]
```diff
@@ -333,6 +336,162 @@ int setNumericEffecterValueHandler(const DBusInterface& dBusIntf,
...
 }
 
+int getEffecterValue(std::string propertyType,
+                     pldm::utils::PropertyValue propertyValue,
+                     uint8_t* effecterValue, size_t effecterValueLength)
+{
+    if (propertyType == "uint32_t")
+    {
+        uint32_t propVal = std::get<uint32_t>(propertyValue);
+#ifdef __STDC_LIB_EXT1__
```
Comment (Pavithrab7) [inline libpldmresponder/platform_numeric_effecter.hpp:346]: why a C style copying, can't we use casting and then return the value?

Comment (jaypadath) [reply libpldmresponder/platform_numeric_effecter.hpp:346]: Please try of std::copy_n works here instead of memcpy

Comment (Pavithrab7) [reply libpldmresponder/platform_numeric_effecter.hpp:346]: please respond to the above comments

Comment (ArchanaKakani) [reply libpldmresponder/platform_numeric_effecter.hpp]: Removed memcpy 

> why a C style copying, can't we use casting and then return the value?

Removed the memcpy and directly passing the effecter value by typecasting it to reinterpret_cast.

Comment (ArchanaKakani) [reply libpldmresponder/platform_numeric_effecter.hpp]: > please respond to the above comments

I have reworked on the code flow for GetNumericEffecter. Now we are not using memcpy. Requesting review again.

**Review by jaypadath — APPROVED** [review]

**Pull request merged**